pkg/registry/graph.go

```diff
@@ -0,0 +1,34 @@
+package registry
+
+type Package struct {
+	Name           string
+	DefaultChannel string
+	Channels       []Channel
+}
+
+type Channel struct {
+	Name            string
+	OperatorBundles []OperatorBundle
+	Head            BundleRef
+}
+
+type OperatorBundle struct {
+	BundlePath      string
+	Version         string // semver string
+	CsvName         string
```

Comment:
WDYT if we replace the first 3 fields with bundleRef? This way we define how we refer to a bundle in one place?